python/cudf/cudf/core/dataframe.py

```diff
@@ -5499,14 +5510,16 @@ def isin(self, values):
     #
     # Stats
     #
-    def _prepare_for_rowwise_op(self):
+    def _prepare_for_rowwise_op(self, method, skipna):
         """Prepare a DataFrame for CuPy-based row-wise operations.
         """
 
-        if any([col.nullable for col in self._columns]):
+        if method not in _cupy_nan_methods_map and any(
+            [col.nullable for col in self._columns]
```

Comment:
I think you can use a generator instead of materializing a list here:
```suggestion
            col.nullable for col in self._columns
```